master/docs/manual/cfg-wwwhooks.rst

```diff
@@ -89,7 +89,7 @@ GitHub hook
 .. note::
 
    There is a standalone HTTP server available for receiving GitHub notifications as well: :contrib-src:`master/contrib/github_buildbot.py`.
-   This script may be useful in cases where you cannot expose the WebStatus for public consumption.
+   This script may be useful in cases where you cannot expose the WebStatus for public consumption. Alternatively, you can setup a reverse proxy :ref:`Reverse_Proxy_Config` 
```

Comment:
well the Reverse_Proxy_Config exposes the whole web status to public.

You need a different config than the one that is described there.

Reply:
so, either it's a documented security warning? or an example config?

Reply:
I think you can add an example here of an nginx config which implements the redirection just for the hook.